QgisModelBaker/qgismodelbaker.py

```diff
@@ -131,6 +134,16 @@ def initGui(self):
             self.tr("Dataset Manager"),
             None,
         )
+        self.__tidmanager_action = QAction(
+            QIcon(
+                os.path.join(
+                    os.path.dirname(__file__),
+                    "images/QgisModelBaker-tidmanager-icon.svg",
```

Comment:
is this file missing or am I missing something??

Reply:
Oh yes. My artistic streak was not active yet :-)